osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -261,7 +263,8 @@ private void load(LocalisationManager localisation)
                                 Margin = new MarginPadding { Top = 10 },
                                 Direction = FillDirection.Horizontal,
                                 AutoSizeAxes = Axes.Both,
-                                Children = getMapper(metadata)
+                                //Child = getMapper(metadata)
+                                Child = string.IsNullOrEmpty(metadata.Author?.Username) ? new LinkFlowContainer() : new MapperLink(metadata.Author, 15)
```

Comment:
* Can use `Empty()` instead of a new link flow. Will need updating test assertions.
* This usage actually has a big problem in that the link doesn't change colour when hovered. This is due to

  https://github.com/ppy/osu/blob/9be3d7a16171ca4dbe15b0ad3689c4f18f35ae2c/osu.Game/Screens/Select/BeatmapInfoWedge.cs#L156

  which means the buffered container won't be redrawn every frame.

  I wouldn't recommend turning it off as it'd probably be a significant performance regression. Instead we might want to consider splitting out the text layer to be above the pre-rendered buffered background of the wedge.